README.markdown

```diff
@@ -316,9 +316,14 @@ Put this into your `$HOME/.zshrc` to call `nvm use` automatically whenever you e
 # place this after nvm initialization!
 autoload -U add-zsh-hook
 load-nvmrc() {
-  if [[ -f .nvmrc && -r .nvmrc ]]; then
-    nvm use
-  elif [[ $(nvm version) != $(nvm version default)  ]]; then
+  local nodeversion="$(nvm version)"
+  local nvmrc_path="$(nvm_find_nvmrc)"
+
+  if [ -n "$nvmrc_path" ]; then
+    if [ "v$(cat $nvmrc_path)" != "$nodeversion" ]; then
```

Comment:
this won't work because the `.nvmrc` file could contain `1` or `v1.0` or `v1.0.0` or `foo` (where "foo" is an alias) - you'd need `$(nvm version "$(cat "${nvmrc_path}")")`